### DISCUSSION TITLE
Noob Question: How do I remove/modify sidebar menu items?

### DISCUSSION THREAD

**Discussion by MakisChristou** [thuliteio/doks#868] [General]
Hello, I recently discovered HUGO and specifically the Doks theme so I am new to all of this. I have successfully added pages to the template as well as modify the existing pages and menus. What I am not able to do as of now is being able to modify the sidebar menu. For example when navigating to the `menus.en.toml` file I can modify the main menu that is visible all the time when loading the page. But once I click one of the items, lets say "Get Started" is one of the default options, I see another menu that I cannot modify or edit in any way (the so called sidebar menu). Is it automatically generated? How do I modify it or delete items in it? Thanks. 

My `menus.en.toml`

```

[[main]]
  name = "Get Started"
  url = "/docs/general/getstarted/"
 weight = 10

[[main]]
  name = "Download"
  url = "/docs/general/download/"
  weight = 10

[[main]]
  name = "Tutorials"
  url = "/blog/"
  weight = 20


[[social]]
  name = "GitHub"
  pre = "<svg xmlns=\"http://www.w3.org/2000/svg\" width=\"20\" height=\"20\" viewBox=\"0 0 24 24\" fill=\"none\" stroke=\"currentColor\" stroke-width=\"2\" stroke-linecap=\"round\" stroke-linejoin=\"round\" class=\"feather feather-github\"><path d=\"M9 19c-5 1.5-5-2.5-7-3m14 6v-3.87a3.37 3.37 0 0 0-.94-2.61c3.14-.35 6.44-1.54 6.44-7A5.44 5.44 0 0 0 20 4.77 5.07 5.07 0 0 0 19.91 1S18.73.65 16 2.48a13.38 13.38 0 0 0-7 0C6.27.65 5.09 1 5.09 1A5.07 5.07 0 0 0 5 4.77a5.44 5.44 0 0 0-1.5 3.78c0 5.42 3.3 6.61 6.44 7A3.37 3.37 0 0 0 9 18.13V22\"></path></svg>"
  url = "https://github.com/MakisChristou/verifit"
  post = "v0.1.0"
  weight = 10

[[footer]]
  name = "Privacy Policy"
  url = "/docs/general/privacypolicy/"
  weight = 10

```

**Comment by h-enk** [comment]
Thanks for using Doks! 

You have the option to let Doks auto-generate the section menu from the directory folder (tree) structure. But, you could also manually generate a sidebar menu. See also https://getdoks.org/docs/how-to/pages-and-menus/menus/#section-navigation
